daily-coding-challenges/24-06-2020/repeatString.js

```diff
@@ -13,7 +13,10 @@
 
 const repeatStringNumTimes = (str, num) => {
 	/* Your implementation goes here */
-	return str;
+	if(num > 0) 
+	return str.repeat(num);
```

Comment:
I like this.. this shows good knowledge of JavaScript APIs.

Reply:
had to read up, thanks to you